aten/src/ATen/native/mkldnn/Conv.cpp

```diff
@@ -124,6 +141,81 @@ Tensor mkldnn_convolution(
   }
 }
 
+Tensor mkldnn_convolution_relu(
+    const Tensor& input,
+    const Tensor& weight,
+    const c10::optional<Tensor>& bias_opt,
+    IntArrayRef stride,
+    IntArrayRef padding,
+    IntArrayRef dilation,
+    int64_t groups) {
+  TORCH_CHECK(input.dim() == 4 || input.dim() == 5,
+      "mkldnn_convolution_relu: currently mkldnn only support 2d or 3d convolution relu fusion");
+  TORCH_CHECK(input.scalar_type() == ScalarType::Float || input.scalar_type() == ScalarType::BFloat16,
+      "mkldnn_convolution_relu: currently mkldnn only support float or bfloat16 input");
+
+  // See [Note: hacky wrapper removal for optional tensor]
+  c10::MaybeOwned<Tensor> bias_maybe_owned = at::borrow_from_optional_tensor(bias_opt);
+  const Tensor& bias = *bias_maybe_owned;
```

Comment:
There's no need to cargo cult the hacky code. Do what the comment tells you to do:

```
/*
 * [Note: hacky wrapper removal for optional tensor]
 *
 * The kernel implementation takes an optional tensor marked in the schema as
 * Tensor? but the C++ function takes Tensor instead of the optional<Tensor>
 * expected by the dispatcher.
 *
 * To remove the hacky wrapper, the C++ function is changed to take
 * optional<Tensor> and unwrap the Tensor value at the beginning of
 * the function, e.g.:
 *   > c10::MaybeOwned<Tensor> weight_maybe_owned =
 *   >     at::borrow_from_optional_tensor(weight_opt);
 *   > const Tensor& weight = *weight_maybe_owned;
 *
 * We may want to make the kernel handle optional directly without
 * going through the creation of a default-constructed Tensor in
 * at::borrow_from_optional_tensor.
 */


```